target/linux/realtek/dts-5.10/rtl9302_zyxel_xgs1250-12.dts

```diff
@@ -0,0 +1,296 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/dts-v1/;
+
+#include "rtl930x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "zyxel,xgs1250-12", "realtek,rtl838x-soc";
+	model = "Zyxel XGS1250-12 Switch";
+
+	// mw.l 0xb8003308 0x00200000 active low toggles PHY reset
```

Comment:
Can you add this GPIO to the MDIO bus or phys that it resets?

Reply:
The PHYs still need to be set up by u-boot on that platform. At present this will not work.

Reply:
Ok, then I think this should be documented as such. Please add a (commented out) reset-gpios property to the relevant node(s), noting why it's not used at the moment.